client/src/components/History/ContentItem/Dataset/DatasetUI.vue

```diff
@@ -49,7 +49,7 @@ either through the props, and make updates through the events -->
 
                 <div class="content-title title p-1 overflow-hidden">
                     <h5 class="text-truncate" v-if="collapsed">
-                        <span class="hid sr-only" data-description="dataset hid">{{ dataset.hid }}</span>
+                        <span class="hid" data-description="dataset hid">{{ dataset.hid }}</span>
```

Comment:
Does the ::after content thing work for you ?

Reply:
Nope, I was just looking into the disappearing Colon actually :)

I noticed the hid disappears when you expand the card, too, since it's a separate element.  I'll follow up on both of these here.